src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/movements/ExternalMovementsSyncService.kt

```diff
@@ -373,6 +375,103 @@ class ExternalMovementsSyncService(
     }
   }
 
+  suspend fun externalMovementChanged(event: ExternalMovementEvent) = when (event.movementType) {
+    "TAP" if event.recordInserted -> externalMovementTapInserted(event)
```

Comment:
For external movements we don't get the usual 3 events (-INSERTED, -UPDATED, -DELETED) so we have to check in the event for recordInserted or recordDeleted.